modules/oadp-installing-dpa-1-2-and-earlier.adoc

```diff
@@ -299,6 +299,7 @@ spec:
 <10> Specify a bucket as the backup storage location. If the bucket is not a dedicated bucket for Velero backups, you must specify a prefix.
 <11> Specify a prefix for Velero backups, for example, `velero`, if the bucket is used for multiple purposes.
 endif::[]
+////
```

Comment:
It is commented out prior to approval to deletion. Am I ok to delete?

Reply:
Since it's only displayed in the deleted assembly due to the `ifdef::virt-installing-configuring-oadp[]` statement, I don't personally see a reason to keep it. But IMO that's up to you/the OADP side